packages/figma-utils/src/variables/generate.ts

```diff
@@ -25,14 +25,20 @@ export type GenerateAsCSSOptions = BaseGenerateOptions & {
  * Generates the given parsed Figma variables into CSS variables.
  *
  * @param data Parsed Figma variables
+ * @param dataDarkTheme Parsed Figma variables for additionally dark theme
  * @param options Optional options to fine-tune the generated output
  * @returns File content of the .css file
  */
-export const generateAsCSS = (data: ParsedVariable, options?: GenerateAsCSSOptions): string => {
+export const generateAsCSS = (
+  data: ParsedVariable,
+  dataDarkTheme?: ParsedVariable,
```

Comment:
Would prefer to move the dataDarkTheme to the options so this is no breaking change for the user.
Same for the SCSS